static/js/info_overlay.js

```diff
@@ -1,16 +1,151 @@
+        effect_html: "(notifies Joe Smith)",
+    },
+    {
+        markdown: "@_**Joe Smith**",
+        usage_html: "(autocompletes from @_joe)",
+        output_html: '<p><span class="user-mention">Joe Smith</span></p>',
+        effect_html: "(links to profile but doesn't notify Joe Smith)",
+    },
+    {
+        markdown: "@**all**",
+        effect_html: "(notifies all recipients)",
+    },
+    {
+        markdown: "#**streamName**",
+        output_html: "<p><a>#streamName</a></p>",
+        effect_html: "(links to a stream)",
+    },
+    {
+        markdown: "/me is busy working",
+        output_html: '<p><span class="sender_name-in-status">Iago</span> is busy working</p>',
+        usage_html: "(send a status message as user Iago)",
+    },
+    {
+        markdown: "Some inline `code`",
+    },
+    {
+        markdown: `\
+\`\`\`
+def zulip():
+    print "Zulip"
+\`\`\``,
+    },
+    {
+        markdown: `\
+\`\`\`python
+def zulip():
+    print "Zulip"
+\`\`\``,
+        output_html: `\
+<div class="codehilite"><pre><span class="k">def</span> <span class="nf">zulip</span><span class="p">():</span>
+    <span class="k">print</span> <span class="s">"Zulip"</span></pre></div>`,
+    },
+    {
+        note_html: i18n.t(
+            'To add syntax highlighting to a multi-line code block, add the language\'s <b>first</b> <a target="_blank" rel="noopener noreferrer" href="https://pygments.org/docs/lexers/">Pygments short name</a> after the first set of back-ticks. You can also make a code block by indenting each line with 4 spaces.',
+        ),
+    },
+    {
+        markdown: "> Quoted",
+    },
+    {
+        markdown: `\
+\`\`\`quote
+Quoted block
+\`\`\``,
+    },
+    {
+        markdown: `\
+\`\`\`spoiler Always visible heading
+This text won't be visible until the user clicks.
+\`\`\``,
+    },
+    {
+        markdown: "Some inline math $$ e^{i \\pi} + 1 = 0 $$",
+    },
+    {
+        markdown: `\
+\`\`\`math
+\\int_{0}^{1} f(x) dx
+\`\`\``,
+    },
+    {
+        note_html: i18n.t(
+            'You can also make <a target="_blank" rel="noopener noreferrer" href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#wiki-tables">tables</a> with this <a target="_blank" rel="noopener noreferrer" href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#wiki-tables">Markdown-ish table syntax</a>.',
+        ),
+    },
+];
+
 export function set_up_toggler() {
+    for (const row of markdown_help_rows) {
+        if (row.markdown && !row.output_html) {
+            const message = {raw_content: row.markdown};
+            markdown.apply_markdown(message);
+            row.output_html = message.content;
+        }
+    }
+
+    $(".informational-overlays .overlay-body").append(render_markdown_help({markdown_help_rows}));
+    $("#markdown-instructions .rendered_markdown").each(function () {
+        rendered_markdown.update_elements($(this));
+    });
```

Comment:
Do we want top call `update_elements` before inserting the object into the DOM?  I think that might be better from a security standpoint.

Reply:
I’d hope `update_elements` isn’t unsafe for user content, but yeah that’s nicer; done.

Reply:
And added `clean_user_content_links` to get `target="_blank"` and friends on the link. (We should probably merge those functions.)

Reply:
Yeah, definitely worth a follow-up PR to merge those.